web/ui/dashboard/src/app/private/components/create-source/create-source.component.ts

```diff
@@ -209,8 +211,9 @@ export class CreateSourceComponent implements OnInit {
 		this.isloading = true;
 		try {
 			const response = this.action === 'update' ? await this.createSourceService.updateSource({ data: sourceData, id: this.sourceId }) : await this.createSourceService.createSource({ sourceData });
-			this.onAction.emit({ action: this.action, data: response.data });
```

Comment:
Why are we no longer emitting the status of this source form?
This is a create source component that has multiple use cases yea?

Or is there something I'm missing?